### PR TITLE
Ft admin set role #187300154

### PR DESCRIPTION
**Pull request by Mukunzijames**
issue in referencing?

Questions:

What does this PR do?

setting role and permission

• setting role for users [admin,seller,buyer]
.creating role 
.assing role to someone

What are the relevant pivotal tracker/Trello stories?
• id of the project
#187300154

### CODE REVIEW

**Review by Ruthuwamahoro — APPROVED** [review]

File: src/database/migrations/20240325172706-create-user.js [hunk 1]
```diff
@@ -20,6 +20,15 @@ module.exports = {
         allowNull: false,
         type: Sequelize.STRING,
       },
+      roleId: {
```
Comment (fab-ryan) [inline src/database/migrations/20240325172706-create-user.js:23]: separate file

File: src/controllers/roleController.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+export const createRole = async (
+  req: Request<object, object, RoleCreationAttributes>,
+  res: Response
+) => {
+  try {
+    const { name, description } = req.body;
+    const roleExist = await Database.Role.findOne({
+      where: {
+        name,
+      },
```
Comment (fab-ryan) [inline src/controllers/roleController.ts]: use trim function

File: src/swagger.json [hunk 3]
```diff
@@ -7,14 +7,15 @@
   },
   "servers": [
     {
-      "url": "http://localhost:3000",
+      "url": "http://localhost:5000",
```
Comment (fab-ryan) [inline src/swagger.json]: please be consistance

**Review by fab-ryan — CHANGES_REQUESTED** [review]
work on those comments

**Review by Nadinefiona — APPROVED** [review]

**Review by fab-ryan — APPROVED** [review]

**Pull request merged**